cortex/serviceconnector.py

```diff
@@ -88,7 +88,7 @@ def post_file(self, uri, files, data, headers=None):
         )
 
     def request_with_retry(
-        self, method, uri, body=None, headers=None, debug=False, **kwargs
```

Comment:
Should be plumb backoff factor as well ? 
Do we get informational messages ( e.g. Failed retrying )
Should we lower the timeout ( 2 x N secs ) ?

^^ Perhaps later improvements ??  (